tests_in_tilt/Tiltfile

```diff
@@ -0,0 +1,51 @@
+# Some common test configurations to easily get up and running with Tilt for Tests
+# (see docs: https://docs.tilt.dev/tests_in_tilt.html)
+
+### GOLANG TESTS ###
+def test_go_all(path, deps, timeout='', tags=None, mod=''):
+    # path: str, deps: List[str], timeout: str='', tags: List[str]=None, mod: str=''
+    if not path.endswith('...'):
+        path = path.rstrip('/')
+        path = path + '/...'
+
+    timeout_str = ''
+    if timeout:  # expects a go-parsable dur
+        timeout_str = '-timeout {}'.format(timeout)
+    tags_str = ''
+    if tags:
+        tags_str = '-tags {}'.format(','.join(tags))
+
+    mod_str = ''
+    if mod:
+        mod_str = '-mod {}'.format(mod)
+
+    cmd = 'go test {mod_str} {tags_str} {timeout_str} {path}'.format(
+        mod_str=mod_str, tags_str=tags_str, timeout_str=timeout_str, path=path)
+    test('test-go-all', cmd, deps=deps)
+
+
+### JAVASCRIPT TESTS ###
+def _test_js_all(pkmanager, dir, deps=None, with_install=False):
+    resource_deps = []
+    if with_install:
+        name = pkmanager + '-install'
+        local_resource(name, 'cd {dir} && {pkmanager} install'.format(
```

Comment:
we should probably have a dep on yarn.lock / package.json here and then ignore them on the test resource

Reply:
yyyyes but also I have no way of knowing where those files live, since they could be in a parent directory. ugh. let me think.